fileio/ft_read_data.m

```diff
@@ -138,6 +140,8 @@
 dataformat      = ft_getopt(varargin, 'dataformat');
 chanunit        = ft_getopt(varargin, 'chanunit');
 timestamp       = ft_getopt(varargin, 'timestamp');
+password        = ft_getopt(varargin, 'password', struct([]));
+sortchannel     = ft_getopt(varargin, 'sortchannel', '');
```

Comment:
looking at the code further down, I see that sortchannel only works for MEF and not for any other file format. That is inconsistent with the documentation and will confuse users

Reply:
Yes, we'll discuss the option of `sortchannel`.